src/schema/users.ts

```diff
@@ -356,6 +361,7 @@ export const typeDefs = /* GraphQL */ `
 
   type ReferralCampaign {
     referredUsersCount: Int!
+    referralCountLimit: Int
```

Comment:
This is optional?

Reply:
I see depends on if it's always queried, but in that case it should be fine like this.